docs/user-manuals/device-scheduling-gpu-share-with-hami.md

```diff
@@ -0,0 +1,154 @@
+# Device Scheduling - GPU Share With HAMi
+
+## Introduction
+
+GPU is an indispensable device for today's large AI model training and inference, and can provide powerful computing power support for AI applications. An NVIDIA A100 80 GB GPU can provide up to 249 times the inference performance on the BERT-LARGE task compared to a CPU. However, behind such powerful computing power is an expensive price. An NVIDIA A100 40GB model GPU chip is priced at around $13,000, while an A100 80GB model GPU chip is priced at around $15,000. We have observed that some inference tasks often use less than a full GPU, but only use, for example, 50% of the computing power or gpu memory. Therefore, sharing a GPU with multiple Pods can significantly improve GPU resource utilization.
+
+[HAMi](https://project-hami.io/docs/developers/hami-core-design/) is a [Cloud Native Computing Foundation](https://cncf.io/) sandbox project which provides the ability to share Heterogeneous AI devices among tasks. Koordinator takes advantage of HAMi's GPU isolation capabilities on the node side to provide an end-to-end GPU sharing solution.
+
+## Setup
+
+### Prerequisite
+
+- Kubernetes >= 1.18
+- Koordinator >= 1.6
+
+### Installation
+
+Please make sure Koordinator components are correctly installed in your cluster. If not, please refer to [Installation](/docs/installation).
+
+#### Runtime Requirements
+
+The scheduled GPU devices are bound to the container requires support from the runtime environment. Currently, there are two solutions to achieve this:
+
+| Runtime Environment                           | Installation                                                 |
+| --------------------------------------------- | ------------------------------------------------------------ |
+| Containerd >= 1.7.0 <br /> Koordinator >= 1.6 | Please make sure NRI is enabled in containerd. If not, please refer to [Enable NRI in Containerd](https://github.com/containerd/containerd/blob/main/docs/NRI.md) |
+| others                                        | Please make sure koord-runtime-proxy component is correctly installed in you cluser. If not, please refer to [Installation Runtime Proxy](installation-runtime-proxy). |
+
+#### HAMi-Core Installation
+
+[HAMi-core](https://github.com/Project-HAMi/HAMi-core) is the in-container gpu resource controller, it operates by Hijacking the API-call between CUDA-Runtime(libcudart.so) and CUDA-Driver(libcuda.so).  Actually, the output of HAMI-core build is 'libvgpu.so', we provide a way to distribute it to Nodes.
```

Comment:
"Actually, the output of HAMI-core build is 'libvgpu.so', we provide a way to distribute it to Nodes."

The output from building HAMi-core is `libvgpu.so`. You can directly install HAMi-core on your nodes by deploying the DaemonSet with the YAML file provided below.